src/dev/flang/be/c/C.java

```diff
@@ -930,7 +932,8 @@ private void createCode(CFile cf, COptions _options) throws IOException
        "#include <errno.h>\n"+
        // defines _O_BINARY
        "#include <fcntl.h>\n"+
-       "#include <stdatomic.h>\n");
+       "#include <stdatomic.h>\n"+
+       "#include <dirent.h>\n");
```

Comment:
Since `dirent` is POSIX the code should be put in `posix.c`. Since recently there are only C-standard headers included here.

Reply:
@maxteufel Yes, of course you have to pull out the relevant code and put it in the `posix.c/win.c`